### DISCUSSION TITLE
Refine Non-Standard Blank Values Data Hygiene Test

### DISCUSSION THREAD

**Discussion by webfanatic** [DataKitchen/dataops-testgen#10] [Ideas]
I see that the Non-Standard Blank Values Anomaly is sometimes too sensitive. I have a column of type BIGINT that has an incrementing ID in it. The Non-Standard Blank Values test is picking up 99, 999, 999999, 9999999 as being Blank Values. It's simple enough to Mute the Anomaly, but a nice to have is to not flag it in the first place. The values in the column are 100% filled and mostly continuous.

**Comment by webfanatic** [comment]
I found a similar, but different example: I have distinct values in my column: 00, 01, all the way up to 12, but the 00 is getting flagged as a NSBV.

**Comment by cbloche** [comment]
Yeah, I agree 99%.  What functional data type are you getting for those columns?  This might help distinguish valid values.

**Comment by webfanatic** [comment]
For the first example, The physical table is VARCHAR(100) (Dataype Suggestion is the same). The Functional Data Type is 'ID-Unique'.

For the seconds one,
Physical: varchar(100)
Datatype Suggestion: INTEGER
Functional Data Type: UNKNOWN
